packages/fe-container/src/styles/_colors.less

```diff
@@ -8,4 +8,5 @@
 :root {
   --microlc-primary-color: #1890FF;
   --microlc-menu-entry-selected-bg-color: #E6F3FF;
+  --error-page-text-color : #0E0E0E;
```

Comment:
This one can be a less variable. 

We should use CSS variables only for froperties that can be set from the configuration. Furthermore, all che CSS variables should have the `microlc` prefix to avoid conflicts with the plugins variables.